### ISSUE TITLE
hacktober2024

### ISSUE THREAD

**Issue by Sumits-Cosmos** [yazdanhaider/Hospital-Management#88]
The project looks interesting, could you assign me the project.

**Comment by yazdanhaider** [comment]
Heyyyy @Sumits-Cosmos I personally checked the application and there is no such issue!!

**Comment by Sumits-Cosmos** [comment]
yeah right, there was some issue with privacy policy.jsx file from my side.
if you can assign me this project, i want to work on some issues which i found.

**Comment by Sumits-Cosmos** [comment]
I want to add a support feature which includes raising tickets, some basic common issues which all user faces.

**Comment by yazdanhaider** [comment]
Go ahead @Sumits-Cosmos
